docker-push.sh

```
@@ -12,7 +12,7 @@ if [[ $1 = "dev" ]]; then
   IMAGE_TAG="$IMAGE_TAG-rc"
 fi;
 
-aws ecr get-login-password --region ap-northeast-2 | docker login --username AWS --password-stdin 291889421067.dkr.ecr.ap-northeast-2.amazonaws.com
+aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/q0g6g7m8
 
-docker buildx build --platform linux/amd64 -t 291889421067.dkr.ecr.ap-northeast-2.amazonaws.com/scc-admin-frontend:$IMAGE_TAG --build-arg="NEXT_PUBLIC_DEPLOY_TYPE=$1" .
-docker push 291889421067.dkr.ecr.ap-northeast-2.amazonaws.com/scc-admin-frontend:$IMAGE_TAG
+docker buildx build --platform linux/amd64 -t public.ecr.aws/i6n1n6v2/scc-admin-frontend:$IMAGE_TAG --build-arg="NEXT_PUBLIC_DEPLOY_TYPE=$1" .
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)